tests/unit/test_modulegraph/test_modulegraph.py

```diff
@@ -290,6 +295,38 @@ def test_find_module(self):
                 self.assertEqual(description, (ext, 'rb', imp.C_EXTENSION))
                 self.assertEqual(fp, None)
 
+            # Module in Package
+            parent = MockNode('mypkg')
+            info = modgraph._find_module('__init__', path=[path, os.path.join(path, 'mypkg')] + sys.path, parent=parent)
```

Comment:
Please add a comment to this test case, explaining what is tested here and why.

One of the reasons such a comment is required: Running `_find_module()` on a module named `__init__` is a flaw, caused be the way modulegraph is implemented. Modulegraph v2 should not have this flaw any more and we will need to adopt this test (and need to know how) :-)

When moving the test to `tests/unit/test_modulegraph_more.py` please consider creating the modules on the fly (see other test in this file), allowing for me detailed tests. I suggest additional tests, verifying that "importing" the *package* works, too.